src/frontend/src/redux/records/actions.ts

```diff
@@ -0,0 +1,33 @@
+import { Dispatch } from 'redux';
+import apiService from '../../service/api-service';
+import { AxiosResponse } from 'axios';
+import { LOAD_SEARCH_RECORDS, LOAD_SEARCH_RECORDS_LOADING } from './types';
+
+export function loadSearchRecords(
+  first_name: string,
+  last_name: string,
+  birthday: string
+): any {
+  return (dispatch: Dispatch) => {
+    dispatch({
+      type: LOAD_SEARCH_RECORDS_LOADING
+    });
+    return apiService(dispatch, {
+      url: '/api/search',
+      data: {
+        first_name: first_name,
+        middle_name: '',
+        last_name: last_name,
+        birth_date: birthday
+      },
+      method: 'post',
+      authenticated: true,
```

Comment:
As of the other PR that removes the Request Middleware, `authenticated: true` no longer does anything so we should remove it.

I didn't think of this when reviewing the other PR, but yeah, we should remove 

```
  authenticated?: boolean;
```

from the Request type def.